src/Action.ts

```diff
@@ -122,10 +129,6 @@ export class Action {
         const tag = this.inputs.tag
         const response = await this.releases.listReleases()
         const releases = response.data
-        if (!releases) {
```

Comment:
What was the rationale for removing this? This is something I previously added to help debug cases where we weren't getting releases here (weird edge cases with Github).

Reply:
Whoopsie, bringing this back!